Libplanet/Net/Swarm.cs

```diff
@@ -759,6 +723,75 @@ private async Task BindingProxies()
             }
         }
 
+        private IAsyncEnumerable<(Peer, Pong)> DialToExistingPeers(
+            CancellationToken cancellationToken)
+        {
+            return new AsyncEnumerable<(Peer, Pong)>(async yield =>
+            {
+                foreach (Peer peer in _peers.Keys)
+                {
+                    try
+                    {
+                        await yield.ReturnAsync(
+                            (peer, await DialPeerAsync(peer, cancellationToken))
+                        );
+                    }
+                    catch (TimeoutException e)
+                    {
+                        _logger.Error(
+                            e,
+                            $"TimeoutException occured ({peer})."
+                        );
+                        continue;
+                    }
+                    catch (IOException e)
+                    {
+                        _logger.Error(
+                            e,
+                            $"IOException occured ({peer})."
+                        );
+                        continue;
+                    }
+                    catch (DifferentAppProtocolVersionException e)
+                    {
+                        _logger.Error(
+                            e,
+                            $"Protocol Version is different ({peer}).");
+                    }
+                }
+            });
+        }
+
+        private async Task SyncPreviousBlocksFromPeers<T>(
```

Comment:
Another comment about naming thing: how about calling this “behind blocks” instead of “previous blocks” here?  For example, GitHub uses the term “behind commits” when comparing two branches.